templates/simple/flake.nix

```diff
@@ -0,0 +1,44 @@
+{
+  nixConfig = {
+    extra-substituters = "https://ocaml.nix-cache.com";
+    extra-trusted-public-keys = "ocaml.nix-cache.com-1:/xI2h2+56rwFfKyyFVbkJSeGqSIYMC/Je+7XXqGKDIY=";
+  };
+
+  inputs = {
+    nixpkgs.url = "github:nix-ocaml/nix-overlays";
+    flake-utils.url = "github:numtide/flake-utils";
+    flake-utils.follows = "nixpkgs/flake-utils";
```

Comment:
I think I prefer `nixpkgs.inputs.flake-utils.follows = "flake-utils";`

Reply:
The problem with that is that if there's a breaking change in `flake-utils` we might break the overlays with that which makes it hard to debug. But I can go either way

Reply:
great point. then let's keep it